.github/workflows/test.yaml

```diff
@@ -58,7 +58,7 @@ jobs:
           skip-pkg-cache: true
 
       - name: Run Go tests 🔬
-        run: go test -cover -covermode atomic -coverprofile=profile.cov -v ./...
+        run: go test -p 1 -cover -covermode atomic -coverprofile=profile.cov -v ./...
```

Comment:
This ensure that tests run in sequence, not parallel, as it causes dead locks and many other issues.